### DISCUSSION TITLE
Component Proposal: Common feedback

### DISCUSSION THREAD

**Discussion by ryanwoldatwork** [uswds/uswds#5773] [Proposals]
### Component Overview

As a website owner,
I want to collect yes/no sentiment on every page on a public website,
So that I can assess usefulness/helpfulness of specific pages.

### Describe the solution you'd like

Desire:

Something like the screenshot below, with USWDS-specific considerations - and perhaps as a component.

![Screenshot 2023-10-16 at 1 28 35 PM](https://github.com/uswds/uswds-proposals/assets/64987852/4f5c1771-9e12-4baa-993c-fc8c2fa7ef56)

### Describe alternatives you've considered

* NPS.gov is using Touchpoints for this
* Other agencies are using other vendors for this too
* Gov.uk example:

<img width="1053" alt="Screenshot 2024-08-15 at 11 42 23 AM" src="https://github.com/user-attachments/assets/30c3ad76-6970-4fcf-9ac3-4c5c77f50856">

<img width="1087" alt="Screenshot 2024-08-15 at 11 42 27 AM" src="https://github.com/user-attachments/assets/e376417b-6207-4a02-a38b-dfb13a4d697d">

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22) for duplicate feature requests.

**Comment by mgifford** [comment]
I do think this would be useful, but right now I think most are implemented with 3rd party JS tools. There are big problems with those generally, but where there are used, they generally aren't managed directly by government.

**Reply by mogul** [reply]
(JFYI: Ryan works on [Touchpoints](https://touchpoints.digital.gov/). 😉)

**Comment by amycole501** [comment]
[Proposal issue started here](https://github.com/uswds/uswds-proposals/issues/35)

**Comment by annepetersen** [comment]
Edited the original proposal above to re-add the screenshot that got moved to the [uswds-proposals repo](https://github.com/uswds/uswds-proposals/issues/35):
![Screenshot 2023-10-16 at 1 28 35 PM](https://github.com/uswds/uswds-proposals/assets/64987852/4f5c1771-9e12-4baa-993c-fc8c2fa7ef56)

**Comment by aklausmeier** [comment]
The VA has a similar feedback flow on the Resources & Support content library. Good/Bad rating which then prompts an additional optional opportunity to provide open text feedback via our standard survey. The Good/Bad rating increases participation significantly.

![Screenshot 2025-05-15 at 9 11 58 AM](https://github.com/user-attachments/assets/01394db3-1a4c-476c-b95a-13ac4e128510)
![Screenshot 2025-05-15 at 9 12 07 AM](https://github.com/user-attachments/assets/1bf6b649-36c5-47bd-884a-e7bd00280ee1)
![Screenshot 2025-05-15 at 9 13 41 AM](https://github.com/user-attachments/assets/82329ac9-77e0-4c5b-b9c5-8ef92dedfae4)
